docs/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,39 @@
+import { memo, useCallback, useState } from 'react';
+import styles from './Accordion.module.scss';
+
+type AccordionProps = {
+  heading: string;
+  isActive: boolean;
+  sections?: string[];
+};
+export const Accordion = memo<AccordionProps>(({ heading, isActive, sections }) => {
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const hasSections = sections && sections.length > 0;
+  const toggleExpanded = useCallback(() => setExpanded((prev) => !prev), []);
+  return (
+    <div className={styles.accordionWrapper}>
+      <header className={styles.accordionHeader} onClick={toggleExpanded}>
+        <h3 className={`${styles.accordionHeading} ${isActive ? styles.accordionHeadingActive : ''}`}>{heading}</h3>
+        {hasSections && (
+          <img
```

Comment:
Semantically speaking it shouldn't be an `img` – chevron should be added with CSS
(+ some aria attributes) https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html